src/components/UserListItemCard/UserListItemCard.tsx

```diff
@@ -5,119 +5,104 @@ import {
   },
-  statusCircle: {
-    height: '8px',
-    width: '8px',
-    marginRight: spacing(1),
-    borderRadius: '50%',
-    display: 'inline-block',
-  },
   cardActions: {
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
     padding: spacing(2),
-    border: `1px solid ${palette.grey[300]}`,
+    borderTop: `1px solid ${palette.grey[300]}`,
   },
-  activeColor: {
-    backgroundColor: '#619a55',
-  },
-  notActiveColor: {
-    backgroundColor: palette.action.disabled,
+  actionsButton: {
+    lineHeight: '1.143',
+    padding: 0,
   },
 }));
 
-export interface PersonCardProps {
-  name: string;
-  organization: string;
-  role: string;
-  active: boolean;
-  personRoute: string;
-  imageSource: string;
+export interface UserListItemCardProps {
+  user?: UserListItemFragment;
+  className?: string;
 }
 
-export const PersonCard: FC<PersonCardProps> = ({
-  name,
-  organization,
-  role,
-  active,
-  personRoute,
-  imageSource,
-}) => {
+export const UserListItemCard: FC<UserListItemCardProps> = ({ user }) => {
   const classes = useStyles();
 
+  const { fullName, id } = user ?? { fullName: '', id: '' };
+
+  // TODO destructure from user when this data becomes queryable
+  const fullNameParts = fullName ? fullName.split(' ') : ['Unknown'];
+  const avatarLetters = fullNameParts.reduce(
+    (initials, part) => initials.concat(part.slice(0, 1).toUpperCase()),
+    ''
+  );
+  const organization = {
+    name: 'Seed Company',
+    role: 'Software Developer',
+  };
+  const { name: organizationName, role } = organization;
+
   return (
     <Card className={classes.personCard}>
       <CardContent className={classes.cardContent}>
         <Avatar
+          loading={!user}
           variant="circle"
-          alt={name}
-          src={imageSource}
           className={classes.personImage}
-        />
+        >
+          {avatarLetters}
+        </Avatar>
         <Typography
           color="textPrimary"
           variant="body2"
           className={classes.personName}
         >
-          {name}
+          {!user ? <Skeleton variant="text" width="10ch" /> : fullName}
         </Typography>
         <Typography
           color="primary"
           variant="caption"
           className={classes.organizationName}
         >
-          {organization}
-        </Typography>
-        <Typography
-          color="textSecondary"
-          variant="caption"
-          className={classes.roleName}
-        >
-          {role}
+          {!user ? <Skeleton variant="text" width="10ch" /> : organizationName}
         </Typography>
         <Typography color="textSecondary" variant="caption">
-          <span
-            className={clsx(classes.statusCircle, {
-              [classes.activeColor]: active,
-              [classes.notActiveColor]: !active,
-            })}
-          />
-          {active ? 'Active' : 'Inactive'}
+          {!user ? <Skeleton variant="text" width="10ch" /> : role}
```

Comment:
Since role names are pretty wide variety of lengths, I think it would be better to have the width be a percentage instead of a certain number of characters. Typically we only use `ch` measurements when we know the actual value is going to be close that number of characters.

(same for org name and full name)

Reply:
The problem here was that the enclosing element has no `width` property; its width is determined by its contents. So setting the `Skeleton` width to a percentage results in a `Skeleton` that's only a few pixels wide

Reply:
I think we've been flexing the cards internally. This one is probably the odd one out since it's so different.

Reply:
I guess that's another good argument for using the landscape-orientation card instead